### ISSUE TITLE
Help to install

### ISSUE THREAD

**Issue by anybody999** [lamhoang1256/netfilm#33]
Hey. Can you help me how to install on localhost or cloudflare?

Thank you.

**Comment by lamhoang1256** [comment]
Source code on this github is old because loklok api down. So project is no longer usable

**Comment by anybody999** [comment]
> Source code on this github is old because loklok api down. So project is no longer usable

But you just update. You are no longer using loklok. Its possible to share current files or guide o install?

Thank you,i really need this.

**Comment by lamhoang1256** [comment]
No, my current source code is using for netfilm.app do not want to public

**Comment by anybody999** [comment]
> No, my current source code is using for netfilm.app. I don't want to public

Can i buy your serviços then? Where we can talk private?

**Comment by anybody999** [comment]
Services*

**Comment by lamhoang1256** [comment]
You can message me with gmail: lamhoang1256@gmail.com
